pkg/services/featuremgmt/registry.go

```diff
@@ -95,6 +97,7 @@ var (
 			Stage:          FeatureStageGeneralAvailability,
 			Owner:          grafanaExploreSquad,
 			AllowSelfServe: true,
+			Expression:     "false",
```

Comment:
Didn't want to change it to `true` in this PR (it will be changed in https://github.com/grafana/grafana/pull/88208)